tools/export/makefile/__init__.py

```diff
@@ -200,7 +200,7 @@ class GccArm(Makefile):
 
     @staticmethod
     def prepare_lib(libname):
-        return "-l:" + libname
+        return "-l" + libname[3:-2]
```

Comment:
Could we check for the `lib` prefix here before we strip it? Don't worry about the `.a` suffix, it's gaurenteed.